Maddy-maddigordon757/Homework-1/fizzbuzz/app.js

```diff
@@ -9,30 +9,33 @@
 
 // ##### Step 1:
 // Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
-
-for (var i=1; i<100; i++)
-{
-    console.log(i);
+for (var i = 1; i <= 100; i++) {
+    if (i % 3 === 0 && i % 5 === 0) {
+        console.log("fizzbuzz")
+        }    
+        else {
```

Comment:
This code should work, but it's cleaner to simply use the`else if` statement rather than nesting another `if / else` construction within an `else` statement.